ddtrace/contrib/unittest/patch.py

```diff
@@ -846,6 +807,8 @@ def handle_cli_run(func, instance: unittest.TestProgram, args: tuple, kwargs: di
             )
             _update_test_skipping_count_span(_CIVisibility._datadog_session_span)
             _finish_span(test_session_span)
+            if _CIVisibility._instance._collect_coverage_enabled and _module_has_dd_coverage_enabled(unittest):
```

Comment:
Calling `_module_has_dd_coverage_enabled()` is redundant since it's being called by `_stop_coverage()` anyway.